app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/coverage/CoverageTab.kt

```diff
@@ -150,6 +151,9 @@ private fun ColumnScope.PerilSection(
   }
 }
 
+val String.color
```

Comment:
Maybe `private val` so that this does not get exposed to a lot of the project as an extension on any string?